beta-pipelines/multiome/atac.wdl

```diff
@@ -0,0 +1,226 @@
+    fastq_gzipped_input_read2: "read 2 fastq file as input for the pipeline, the cellular barcodes must be the first part of the read name separated by colon"
+    min_length: "minimum length for trimming. Reads that are too short even before adapter removal are also discarded"
+    quality_cutoff: "cutadapt option to trim low-quality ends from reads before adapter removal"
+    adapter_seq_read1: "cutadapt option for the sequence adapter for read 1 fastq"
+    adapter_seq_read2: "cutadapt option for the sequence adapter for read 2 fastq"
+    tar_bwa_reference: "the pre built tar file containing the reference fasta and corresponding reference files for the BWA aligner"
+    read_group_id: "the read group id to be added upon alignment"
+    read_group_sample_name: "the read group sample to be added upon alignment"
+    output_base_name: "base name to be used for the pipelines output and intermediate files"
+    monitoring_script : "script to monitor resource comsumption of tasks"
+    bwa_threads: "the number of threads to use during bwa alignment"
+    bwa_disk_size : "disk size used in bwa alignment step"
+    bwa_mem_size : "the size of memory used during bwa alignment"
+    trim_disk_size : "disk size used in trimming adapters step"
+    trim_mem_size: "the size of memory used during trimming adapters step"
+  }
+
+  call TrimAdapters {
+    input:
+      fastq_input_read1 = fastq_gzipped_input_read1,
+      fastq_input_read2 = fastq_gzipped_input_read2,
+      min_length = min_length,
+      quality_cutoff = quality_cutoff,
+      adapter_seq_read1 = adapter_seq_read1,
+      adapter_seq_read2 = adapter_seq_read2,
+      output_base_name = output_base_name,
+      mem_size = trim_mem_size, 
+      disk_size = trim_disk_size, 
+      monitoring_script = monitoring_script
+  }
+
+  call BWAPairedEndAlignment {
+    input:
+      fastq_input_read1 = TrimAdapters.fastq_trimmed_adapter_output_read1,
+      fastq_input_read2 = TrimAdapters.fastq_trimmed_adapter_output_read2,
+      tar_bwa_reference = tar_bwa_reference,
+      read_group_id = read_group_id,
+      read_group_sample_name = read_group_sample_name,
+      nthreads = bwa_threads,
+      mem_size = bwa_mem_size,
+      disk_size = bwa_disk_size,
+      output_base_name = output_base_name,
+      monitoring_script = monitoring_script
+    }
+}
+  # trim read 1 and read 2 adapter sequeunce with cutadapt
+  task TrimAdapters {
+    input {
+      File fastq_input_read1
+      File fastq_input_read2
+      Int min_length
+      Int quality_cutoff
+      String adapter_seq_read1
+      String adapter_seq_read2
+      String output_base_name
+      String docker_image = "quay.io/broadinstitute/cutadapt:1.18"
+      File monitoring_script
+      Int? disk_size = ceil(2 * ( size(fastq_input_read1, "GiB") + size(fastq_input_read2, "GiB") )) + 200
```

Comment:
this doesn't need to be optional since you are setting it to a default
```suggestion
      Int disk_size = ceil(2 * ( size(fastq_input_read1, "GiB") + size(fastq_input_read2, "GiB") )) + 200
```

Reply:
Makes sense, just changed it!